File: 561.ArrayPartitionI/sum.py
class Solution(object):
    def arrayPairSum(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        nums.sort()
        return sum(nums[::2])


if __name__ == '__main__':
    nums = [1,4,3,2]
    print(Solution().arrayPairSum(nums))

File: 561.ArrayPartitionI/sum.cc
class Solution {
public:
    int arrayPairSum(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int sum = 0;
        for (int i = 0; i < int(nums.size()); i += 2) {
            sum += nums[i];
        }
        return sum;
    }
};
